gitwatch.go

```diff
@@ -5,21 +5,29 @@ package gitwatch
 
 import (
 	"context"
+	"fmt"
+	"io"
 	"net/url"
 	"path/filepath"
 	"strings"
 	"time"
-	"io"
 
-	"golang.org/x/xerrors"
 	"github.com/pkg/errors"
+	"golang.org/x/xerrors"
 	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
 	"gopkg.in/src-d/go-git.v4/plumbing/transport"
 )
 
+// Repository provides a url and branch configuration
```

Comment:
"Repository represents a Git repository address and branch name" would describe it better, just in case future contributors want to know what's what.